### DISCUSSION TITLE
Mac OS Big Sur thinks Blackhole is a malware... How do I disable the warning?

### DISCUSSION THREAD

**Discussion by SimLeveq** [ExistentialAudio/BlackHole#640] [Q&A] [answered]
Hi there, I just updated my mac to Mac OS Big Sur (11.6.8).

Now, every time I boot, I have a notification saying _"BlackHole.driver" will damage your computer. You should move it to the Trash."_

Soooo... Thanks Apple, but no. The thing is, there's nowhere to check a box to "stop warning about this next time"...  It's annoying, how can I tell my computer to stop warning me about this every time I boot?

On system preferences, I see nothing relevant in "security" tab. 
I tried reinstalling to the latest version. 
I can't find anything in the forum... (is it because everybody figured this out??)

Maybe there's something to do in the terminal or something? I'm not that much a computer wiz, so if there's clear step by step to unable this warning, I would be very grateful!

Thanks in advance! 

EDIT : 
Ok nevermind, I used the uninstallers, (rebooted and still got the popup), then put everything in trash and emptied it, rebooted, and no popup. Installed the latest version and now everything is fine. 

Sorry and have a great day.

**Comment by KingAab001** [comment]
I rested my macOS, but it didn't work, can you tell me what did you do?

**Reply by SimLeveq** [reply]
Yeah me too at first, I tried to only delete Blackhole in the HAL folder, and it didn't work, even when I rebooted my computer. Even if I re-installed from the latest installer.

So : 

I used the uninstallers : https://github.com/ExistentialAudio/BlackHole#option-1-use-uninstaller. The 3 of them just to be sure, even though i haven't installed the 64ch one.

And then deleted everything Blackhole related, even in my downloads folder, and emptied the trash.

When I restarted my computer, the warning was gone. 

I then downloaded the latest installer, and installed it : https://github.com/ExistentialAudio/BlackHole#option-1-download-installer

And now everything is fine!

**Comment by orchetect** [comment, marked as answer]
Answered above: https://github.com/ExistentialAudio/BlackHole/discussions/640#discussioncomment-3318784

**Comment by devinroth** [comment]
Looks like you have malware.

https://geeksadvice.com/remove-will-damage-your-computer-you-should-move-it-to-trash-pop-up-mac/

**Reply by SimLeveq** [reply]
No I haven't. My computer thought Blackhole was a malware, but it isn't, right? _That_ was the issue. 

And, the issue was resolved, as you may have read the rest of the post.

Cheers
